enterprise/backup/src/main/java/org/neo4j/backup/OnlineBackup.java

```diff
@@ -302,7 +302,7 @@ public OnlineBackup full( String targetDirectory, boolean verification, Config t
     public OnlineBackup incremental( String targetDirectory )
     {
         outcome = new BackupService().doIncrementalBackup( hostNameOrIp, port, new File( targetDirectory ),
-                timeoutMillis, defaultConfig() );
+                getConsistencyCheck( true ), timeoutMillis, defaultConfig() );
```

Comment:
Right i think it's fine to do that since it's not a patch fix release and by default it's kind of reasonable to have consistent backup, also if you really want to avoid check - if that taking to long you have a way to disable it, if you still using this deprecated api.